org.lflang/src/org/lflang/TargetProperty.java

```diff
@@ -227,15 +217,15 @@ public enum TargetProperty {
             (config, value, err) -> {
                 config.fileNames.addAll(ASTUtils.elementToListOfStrings(value));
             }),
-    
+
     /**
      * Flags to be passed on to the target compiler.
      */
     FLAGS("flags", UnionType.STRING_OR_STRING_ARRAY,
-            Arrays.asList(Target.C, Target.CCPP), (config, value, err) -> {
-                config.compilerFlags = ASTUtils.elementToListOfStrings(value);
-            }),
-    
+        Arrays.asList(Target.C, Target.CCPP), (config, value, err) -> {
```

Comment:
If we use Cmake, we don't know which compiler gets used. Then how do we know which flags to use? Shouldn't we also get rid of this option then?

Reply:
Yes, I think it is always dangerous and discouraged to use compiler flags with CMake. However, they can be a handy workaround when platform-independent strategies fail, so I am inclined to keep the option here.

Reply:
Currently we issue a warning, not an error, when you use compiler flags with CMake. It seems reasonable to stick with that policy.